docs/sql/kill.txt

```diff
@@ -0,0 +1,37 @@
+.. highlight:: psql
+.. _kill_job:
+
+=========
+Kill Jobs
+=========
+
+each executed sql statement results in a corresponding job.
+Jobs that are currently executed are logged in the system table ``sys.jobs`` 
+(see :ref:`jobs_operations_logs`).
```

Comment:
To obtain the uuid of a job you need to enable stats (see ref).
To prevent job logging, set the queue size to zero.